src/components/hero/campaignHero.tsx

```diff
@@ -4,9 +4,19 @@ import { useEffect, useState } from "react";
 import CampaignHeroSubText from "./campaignHeroSubText";
 import CampaignHeroHeading from "./campaignHeroHeading";
 import CampaignHeroLoading from "./campaignHeroLoading";
+import defaultFeaturedItem from "../../data/defaultFeaturedItemData";
+
+import appConfig from "appConfig";
 
 const CampaignHero = ({ imageID }) => {
+  const defaultFeaturedItemResponse =
+    defaultFeaturedItem[appConfig["environment"]];
+  console.log("defaultItem is:", defaultFeaturedItemResponse);
+
   const [data, setData] = useState<any>({});
```

Comment:
Here is another place where the `any` can be replaced with the `FeaturedItemData` interface.